Day22/Solution.swift

```diff
@@ -0,0 +1,8 @@
+func getHeight(root: Node?) -> Int {
+    	if root == nil || (root?.left == nil && root?.right == nil) {
```

Comment:
for this request you to update with below solution: 
```
func getHeight(root: Node?) -> Int {
    return root == nil ? -1 : 1 + max(getHeight(root: root!.left), getHeight(root: root!.right))
}
```

Reply:
Done!